js/myparcel.js

```diff
@@ -50,79 +76,78 @@ MyParcel = {
 			MyParcel.exportDeliveryOptionToWebshop();
 		});
 
-		$('#mypa-signature-selector').on('change', function(e)
-		{
-			MyParcel.toggleDeliveryOptions();
-		});
-
-		$('#mypa-recipient-only-selector').on('change', function()
-		{
-			MyParcel.toggleDeliveryOptions();
-		});
-
+		//show default delivery options and hide PostNL options
```

Comment:
Geen `// bla`, maar `/* bli */`
Als er een plugin aanwezig is die dit minified, komt alles op 1 regel te staan. Dan is alles na deze regel een comment.
Zie ook andere plekken

Reply:
Done!